Error when execute ´npm run build-assets´

I try to install de application and I have a problem when execute 

**npm run build-assets**

**Server Info:**
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.3 LTS
Release:        16.04
Codename:       xenial

node -v
v9.3.0

npm -v
5.5.1


**The error is:**
npm run build-assets

> saleor@0.0.0 build-assets /home/ubuntu/ecommerce_project/ecommerce
> node ./node_modules/webpack/bin/webpack.js -p

events.js:136
      throw er; // Unhandled 'error' event
      ^

Error: write EPIPE
    at _errnoException (util.js:999:13)
    at WriteWrap.afterWrite [as oncomplete] (net.js:883:14)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! saleor@0.0.0 build-assets: `node ./node_modules/webpack/bin/webpack.js -p`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the saleor@0.0.0 build-assets script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2018-01-04T21_30_43_099Z-debug.log

Any Idea?
Thanks!

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "eslint-plugin-standard": "^3.0.1",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
-    "node-sass": "^4.3.0",
+    "node-sass": "^4.7.2",
     "postcss": "^6.0.11",
     "postcss-loader": "^2.0.6",
     "sass-loader": "^6.0.6",

